blog/authors.yml

```diff
@@ -0,0 +1,23 @@
+afflom:
+  name: Alex Flom
+  title: Docusaurus maintainer
+  url: https://github.com/afflom
+  image_url: https://github.com/afflom.png
+
+jpower432:
+  name: Jennifer Power
+  title: Docusaurus maintainer
+  url: https://github.com/jpower432
+  image_url: https://github.com/jpower432.png
+
+sabre1041:
+  name: Andrew Block
+  title: Docusaurus maintainer
+  url: https://github.com/sabre1041
+  image_url: https://github.com/sabre1041.png
+
+usrbinkat:
+  name: Kat Morgan
+  title: Developer Advocate
```

Comment:
you can be whatever you want in this new world!

Reply:
I am who I am